providers/ZoraCreateProvider.tsx

```diff
@@ -1,24 +1,77 @@
 'use client'
 
-import useZoraCreate from '@/hooks/useZoraCreate'
-import React, { createContext, useContext, useMemo } from 'react'
+import React, { createContext, useContext, useState } from 'react'
+import { useAccount, usePublicClient, useSwitchChain } from 'wagmi'
+import { useWriteContracts } from 'wagmi/experimental'
+import { createCreatorClient } from '@zoralabs/protocol-sdk'
+import { CHAIN_ID, REFERRAL_RECIPIENT } from '@/lib/consts'
+import { usePaymasterProvider } from '@/providers/PaymasterProvider'
+import useCreateSuccess from '@/hooks/useCreateSuccess'
+import useConnectWallet from '@/hooks/useConnectWallet'
+import getSalesConfig from '@/lib/zora/getSalesConfig'
+import useCreateMetadata from '@/hooks/useCreateMetadata'
+import { toast } from 'react-toastify'
 
 const ZoraCreateContext = createContext(undefined)
 
 const ZoraCreateProvider = ({ children }: any) => {
-  const zoraCreate = useZoraCreate()
+  const publicClient = usePublicClient()!
+  const { address } = useAccount()
+  const { getCapabilities } = usePaymasterProvider()
+  const { data: callsStatusId, writeContractsAsync } = useWriteContracts()
 
-  const value = useMemo(() => ({ ...zoraCreate }), [zoraCreate])
+  const { connectWallet } = useConnectWallet()
+  const createMetadata = useCreateMetadata()
+  const { switchChain } = useSwitchChain()
+  const [creating, setCreating] = useState<boolean>(false)
 
-  return <ZoraCreateContext.Provider value={value}>{children}</ZoraCreateContext.Provider>
+  useCreateSuccess(callsStatusId, () => setCreating(false))
+
+  const create = async (chainId = CHAIN_ID) => {
```

Comment:
expect `provider` to be `minimal`
- actual: functions / state defined in `provider`
- expected: functions / state defined in `hooks`